content/en/security/code_security/static_analysis/setup/_index.md

```diff
@@ -52,44 +52,85 @@ If you are using another source code management provider, configure Static Code
-| `ignore`           | A list of path prefixes and glob patterns to ignore for this specific ruleset. Matching files will not be analyzed.  | `false`  |
-| `only`             | A list of path prefixes and glob patterns to analyze for this specific ruleset. Only matching files will be analyzed.| `false`  |
+There are three levels of configuration:
 
-You can include the following **rule** options in the `static-analysis.datadog.yml` file:
+* Org Level Configuration (Datadog)
+* Repo Level Configuration (Datadog)
+* Repo Level Configuration (Repo File)
 
-| Name               | Description                                                                                                          | Required |
-|--------------------|----------------------------------------------------------------------------------------------------------------------|----------|
-| `ignore`           | A list of path prefixes and glob patterns to ignore for this specific rule. Matching files will not be analyzed.     | `false`  |
-| `only`             | A list of path prefixes and glob patterns to analyze for this specific rule. Only matching files will be analyzed.   | `false`  |
-| `arguments`        | A map of values for rules that support customizable arguments.                                                       | `false`  |
+All three locations use the same YAML format for configuration. These configurations are merged **in order** using an overlay/patch merge method. For example, lets look at these two sample YAML files:
 
-The map in the `arguments` field uses an argument's name as its key, and the values are either strings or maps:
+```yaml
+rulesets:
+ - A
+   rules:
+      foo:
+        ignore: ["**"]
+        args: ["my_arg1", "my_arg2"]
+```
 
-* To set a value for the whole repository, you can specify it as a string.
-* To set different values for different subtrees in the repository, you can specify them as a map from a subtree prefix to the value that the argument will have within that subtree.
+```yaml
+rulesets:
+ - A
+    rules:
+        foo:
+            ignore: ["my_ignored_file.file"]
+        bar:
+            only: ["the_only_file.file"]
+ - B
+   
+```
+
+The merge of these YAML files with a overlay/patch method would be the following, if we merged in order, first file with the second
+
+```yaml
+rulesets:
+ - A
+    rules:
+        foo:
+            ignore: ["my_ignored_file.file"]
+            args: ["my_arg1", "my_arg2"]
+        bar:
+            only: ["the_only_file.file"]
+ - B
+   
+
+```
 
-The full structure of the `static-analysis.datadog.yml` file is as follows:
+As you can see, the `ignore: ["**"]` from the first file has been overlayed with the `ignore: ["my_ignored_file.file"]` since the second file's value takes precedence in the event of a conflict due to merge order. The `args` field from the first file is retained because there is no conflicting value in the second file.
```

Comment:
```suggestion
As you can see, the `ignore: ["**"]` from the first file was overlayed with the `ignore: ["my_ignored_file.file"]`. This happened because there was a conflict and the second file's value took precedence due to merge order. The `args` field from the first file is retained because there is no conflicting value in the second file.
```